src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/runScript.jelly

```diff
@@ -122,7 +122,7 @@ THE SOFTWARE.
 				<div align="right">
 					<f:submit value="${%Run}" name="run" />
 				</div>
-				<script>$('script').focus();</script>
+				<script>document.getElementById('script').focus();</script>
```

Comment:
Amends #58. Automated testing exposed this lingering use of Prototype, now removed in recent releases of Jenkins core.